repos/aspnetcore-tooling.proj

```diff
@@ -1,9 +1,4 @@
 <Project>
-
-  <PropertyGroup>
-    <SourceDirectory>AspNetCore-Tooling</SourceDirectory>
```

Comment:
Remove this project entirely? It no longer contributes to the .NET SDK in 5.0, it's just VS and VS Code stuff. https://github.com/dotnet/versions/pull/636#issuecomment-684083154 (and I saw this elsewhere too, e.g. it's not in the arcade-powered source-build plan).

Reply:
Removed.